tools/c7n_gcp/tests/test_gke.py

```diff
@@ -67,16 +73,46 @@ def test_cluster_get(self):
             ],
         )
 
+    @pytest.mark.skip(reason="Need to sort out perms")
```

Comment:
Nice use of `skip` here to show progress/intent, very helpful as a reader :+1:

Reply:
Thanks, until we can get a clean recording, that's about the only thing we can do - otherwise the test will fail.